Rakefile

```diff
@@ -99,57 +120,62 @@ namespace :ci do
 end
 
 namespace :docs do
-  desc "Generate markdown docs for Bolt's transport configuration options"
+  desc 'Generate markdown docs for bolt.yaml'
```

Comment:
These feels like the ultimate meta-programming challenge....but I think having it be verbose is more clear and less brittle.